lib/takeaway.rb

```diff
@@ -0,0 +1,37 @@
+class Menu
+  DISHES = {:burger => 5, :hotdog => 4, :fries => 3, :milkshake => 4, :soda => 3}
+
+  attr_reader :dishes, :basket, :basket_total
+
+  def initialize
+    @dishes = DISHES
+    @basket = []
+    @basket_total = 0
+  end
+
+  def list_dishes
+    dishes.each {|item, price| puts "#{item.to_s.capitalize} £#{price}"}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Do not write to stdout. Use Rails' logger if you want to log.